ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -382,6 +386,9 @@ def test_function(
             test_compile=test_flags.test_compile,
             **kwargs,
         )
+        assert ivy.nested_map(
+            ret_from_target, lambda x: ivy.is_ivy_array(x) if ivy.is_array(x) else True
```

Comment:
We shouldn't use the global ivy, we should use the local version from the context manager `gt_backend`